src/common/options.cc

```diff
@@ -776,6 +776,11 @@ std::vector<Option> get_global_options() {
     .set_default(5)
     .set_description(""),
 
+    // enable qat acceleration support for compression
```

Comment:
@qwren  please put this as the description of this option, like
```c++
.set_description("enable qat acceleration support for compression"),
```

Reply:
Yes. Got it!